examples/with_bevy/src/main.rs

```diff
@@ -78,11 +82,11 @@ fn render_scenes(
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
-        .add_plugin(VelloPlugin)
+        .add_plugins(VelloPlugin)
         .add_systems(Startup, setup)
         .add_systems(Update, bevy::window::close_on_esc)
         .add_systems(Update, cube_rotator_system)
-        .add_plugin(ExtractComponentPlugin::<VelloScene>::default())
+        .add_plugins(ExtractComponentPlugin::<VelloScene>::default())
         .add_systems(Update, render_fragment)
```

Comment:
Should these disjoint calls be combined? I.e. `add_plugins((DefaultPlugins, VelloPlugin, Extract...))`. I'm not sure what's currently idiomatic.

Same with the (non-startup) `add_systems` calls, fwiw

Reply:
I think in samples that I saw, it seemed to go either way ... we should reach out to the people doing the bevy /vello integration and suggest that they look at how we can improve this as well. :)

Going to land it as-is for now though.